docs/content/en/docs/getting-started/metrics/_index.md

```diff
@@ -4,3 +4,272 @@ description: Learn how Keptn metrics enhances your deployment
+as well as the Kubernetes CLI.
+
+### Define metrics providers
+
+Populate a
+[KeptnMetricsProvider](../../yaml-crd-ref/metricsprovider.md)
+resource for each external observability platform you want to use.
+
+For our example, we define two observability platforms:
+
+- `dev-prometheus`
+- `dev-dynatrace`
+
+You can specify a virtually unlimited number of providers,
+including multiple instances of each observability platform.
+Each one must be assigned a unique name,
+identified by the type of platform it is
+and the URL.
+
+> Note: The video and example application use an older syntax
+  of the `KeptnMetricsProvider` and `KeptnMetric` resources.
+  The syntax shown in this document is correct for v0.7.1 and later.
+
+Definition of
+[dev-prometheus](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/keptn-prometheus-provider.yaml)
+data source:
+
+```yaml
+kind: KeptnMetricsProvider
+metadata:
+  name: dev-prometheus
+  namespace: simplenode-dev
+spec:
+  type: prometheus
+  targetserver: "http://prometheus-k8s-monitoring-svc.cluster.local:9090"
+```
+
+Definition of the
+[dev-dynatrace](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/dynatrace-provider.yaml.tmp)
+data source.
+Note that the `dev-dynatrace` server is protected by a secret key
+so that information is included in the provider definition:
+
+```yaml
+kind: KeptnMetricsProvider
+metadata:
+  name: dev-dynatrace
+  namespace: simplenode-dev
+spec:
+  type: dynatrace
+  targetServer: "https://hci34192.live.dynatrace.com"
+  secretKeyRef
+    name: dynatrace
+    key: DT_TOKEN
+...
+```
+
+### Define KeptnMetric information
+
+The [KeptnMetric](../../yaml-crd-ref/metric.md) resource
+defines the information you want to gather,
+specified as a query for the particular observability platform
+you are using.
+You can define any type of metric from any data source.
+
+In our example, we define two bits of information to retrieve:
+
+- Number of CPUs, derived from the `dev-prometheus` data platform
+- `availability` SLO, derived from the `dev-dynatrace` data platform
+
+Each of these are configured to fetch data every 10 seconds
+but you could configure a different `fetchIntervalSeconds` value
+for each metric.
+
+The
+[keptn-metric.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/keptn-metric.yaml)
+file for our example looks like:
+
+```yaml
+apiVersion: metrics.keptn.sh/v1alpha2
+kind: Keptnmetric
+metadata:
+  name: available-cpus
+  namespace: simplenode-dev
+spec:
+  provider:
+    name: dev-prometheus
+  query: "sum(kube_node_status_cvapacity{resources`cpu`})"
+  fetchIntervalSeconds: 10
+...
+apiVersion: metrics.keptn.sh/v1alpha2
+kind: Keptnmetric
+metadata:
+  name: availability-slo
+  namespace: simplenode-dev
+spec:
+  provider:
+    name: dev-dynatrace
+  query: "func:slo.availability_simplenodeservice"
+  fetchIntervalSeconds: 10
+...
```

Comment:
to stay consistent with the yaml syntax, we should use `---` to separate the two KeptmMetric objects - this way this can be directly copied into a file and be applied